Terminal.Gui/Core.cs

```diff
@@ -1779,7 +1881,7 @@ public override bool MouseEvent (MouseEvent mouseEvent)
 
 			int nx, ny;
 			if ((mouseEvent.Flags == (MouseFlags.Button1Pressed | MouseFlags.ReportMousePosition) ||
-				mouseEvent.Flags == MouseFlags.Button4Pressed)) {
+				mouseEvent.Flags == MouseFlags.Button3Pressed)) {
 				if (dragPosition.HasValue) {
```

Comment:
I do not understand this change.

Reply:
For the rightmost mouse button in Windows Driver is defined the Button 4 but in the Unix is defined the Button 3. So all the code in the drag where if the user click with the right mouse button in Windows the drag work, but not in Unix that uses the Button 3. It's not working in the Unix yet because limitations of bitwise flags combinations.

Reply:
Done